### ISSUE TITLE
The MRTK 2.4.0 does not work as described.

### ISSUE THREAD

**Issue by laultman** [MicrosoftDocs/mixed-reality#2405]
MRTK 2.4.0
Several attempts completely starting over and using several different versions of Unity 2019.3.15f1, 2019.4.4f1, and 2020.1.0f1 all produce the same result - a mesh screen only.
Expected: "As you walk around, you should see a spatial mapping mesh covering the surfaces that are perceived by the HoloLens. Additionally, you should see indicators on your hands and fingers for hand tracking and a frame rate counter for keeping an eye on app performance"
None of the expected indicators work. This documentation clearly has not tested operation with the HoloLens 2 nor does the Unity test screen work. The only success is getting it to load onto the HoloLens 2 from Visual Studio 2019 16.6.5
I have tried the other parts of the HoloLens 2 tutorial. None work as expected. MRTK is severely flawed.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 62d386bb-3f93-d434-e7b3-28c58ea673e9
* Version Independent ID: a1a9b02f-66a0-feb0-7a08-9a8d2cf398d1
* Content: [Getting started tutorials - 2. Initializing your project and deploying your first application - Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/mr-learning-base-02)
* Content Source: [mixed-reality-docs/mr-learning-base-02.md](https://github.com/MicrosoftDocs/mixed-reality/blob/master/mixed-reality-docs/mr-learning-base-02.md)
* Product: **mixed-reality**
* Technology: **mr-dev**
* GitHub Login: @jessemcculloch
* Microsoft Alias: **jemccull**

**Comment by hferrone** [comment]
Hi @laultman , could you please provide some clarity on the other MRTK issues you opened so our team can assist?

**Comment by laultman** [comment]
@hferrone The list of items that either do not work outright or are so poorly described in the documentation that a person new to the HoloLens/Unity development can't make it work indicates the severity of the problem. I have 30+ years in C, C++, and C# can't make it work. "Can't make it work" means two things, 1- the documentation/tutorials does not make it clear enough what the outcome should be for a person with reasonable skill to know what to expect so therefore something as simple as moving your hands does nothing and you see no action - its not working. 2- When the whole point of a tutorial is to cause some action and this does not happen then there is a problem either with the documentation or the tool is buggy. For example the DirectionalIndicator in MRTK simply does not work in Unity or in HoloLens.
I have found so many items that don't work that as expected or described as to make the MRTK untrustworthy. My hope is the documentation is lacking, I would hate to try and deliver to a customer something as unreliable as I have seen so far.
The problems are so great that I am now keeping a track of all iterations to identify problems and workarounds. I am in my 8th work through of the HoloLens 2 tutorials in the Microsoft Documentation. Currently I am at Tutorial 6, and have stopped at the adding Text section. Multiple problems exist in both the Unity play mode and when deployed to HoloLens 2. For example the placement of the Menu is out of view in play or HoloLens 2 run modes as built. I had to move the camera to bring it into view, then the actions described did not function (the buttons cannot be accessed or the TapToPlace action fails). I suspect the TapToPlace is failing because the object cannot be accessed in HoloLens and can only be accessed in Unity play mode.
It is my opinion that your documentation and code are far out of alignment preventing meaningful use of HoloLens. You therefore completely fail to achieve the very purpose the toolkit claims to provide: "MRTK-Unity is a Microsoft-driven project that provides a set of components and features, **used to accelerate cross-platform MR app development in Unity**."
So hopefully you might see that the documentation may intend that an action only be tested in Unity play - it just does not make it clear if that is the case or not. I expect the activity in Unity play to exactly match the action in HoloLens because that is my intended environment. (FYI the XR Plugin is no good.)
